tool-base/src/main/java/io/spine/tools/java/fs/SourceFile.java

```diff
@@ -60,31 +65,31 @@ static SourceFile of(Path path) {
     }
 
     /**
-     * Obtains the generated file {@link Path} for the specified type.
+     * Obtains a references to the generated Java file of the specified type.
      *
      * @param type
      *         the type from which the file is generated
-     * @return a relative file path
+     * @return a source file with a relative path
      */
-    @SuppressWarnings("unused")
     public static SourceFile forType(Type<?, ?> type) {
+        checkNotNull(type);
         var classFile = whichDeclares(type.javaClassName());
         return classFile;
     }
 
     /**
-     * Resolves the file which contains the declaration of the given class.
+     * Resolves a file which contains the declaration of the given class.
      *
-     * <p>The resulting {@code SourceFile} represents a <strong>relative</strong> path to the Java
-     * file starting at the top level package.
+     * <p>The resulting instance represents a <strong>relative</strong> path
+     * to the Java file starting at the top level package.
```

Comment:
Once we are here, can we please have an example here? It looks like there may be several options on how such a path looks like. I.e., starting with `./`, or with `/`, or with nothing but the first sub-package name, etc.

Reply:
Added to the top-level Javadoc.